src/Package/Impl/DataInspect/VariableView.xaml.cs

```diff
@@ -152,5 +154,11 @@ public partial class VariableView : UserControl, IDisposable {
                 ew.ShowDetailCommand.Execute(ew);
             }
         }
+
+        private Task DeleteCurrentVariableAsync() {
+            var node = RootTreeGrid.SelectedItem as ObservableTreeNode;
+            var ew = node?.Model?.Content as VariableViewModel;
+            return ew?.DeleteAsync();
```

Comment:
method either should be async or `return ew?.DeleteAsync() ?? Task.CompletedTask`